packages/@lightningjs/ui-components/src/components/ListItem/ListItemPicker.d.ts

```diff
@@ -30,10 +30,41 @@ export type ListItemPickerStyle = ListItemStyle & {
   iconRightSrc: string;
 };
 
-export default class ListItemPicker extends ListItem {
-  options?: Record<string, unknown>;
+declare namespace ListItemPicker {
+  export interface TemplateSpec extends ListItem.TemplateSpec {
+    /**
+     * list of selectable options
+     */
+    options?: string[];
+
+    /**
+     * index of current selected option
+     */
+    selectedIndex?: number;
+
+    style?: StylePartial<ListItemStyle>;
+  }
+}
+
+declare class ListItemPicker<
+  TemplateSpec extends ListItemPicker.TemplateSpec = ListItemPicker.TemplateSpec,
+  TypeConfig extends lng.Component.TypeConfig = lng.Component.TypeConfig
+> extends ListItem<TemplateSpec, TypeConfig> {
+  /**
+   * list of selectable options
+   */
+  options?: string[];
+
+  /**
+   * index of current selected option
+   */
   selectedIndex?: number;
-  get selectedOption(): Record<string, unknown>; // TODO is this correct?
+
+  /**
+   * returns the item from the `options` array at the current `selectedIndex`
+   */
+  get selectedOption(): string; // TODO is this correct?
```

Comment:
This looks correct to me.

Reply:
if it's a public method(not underscored) it has to be included. the thing that's getting flagged here I think, is whether or not that return type is correct

Reply:
return type 'string' for the selectedOption is correct, since options is an array of strings and selectedOption will retrieve a single selected item from the options.
 Does that make sense?

Reply:
Looks like it's actually an object that represents the whole item being selected. Not sure if that's what we want as the result of that getter or if we wanted something more narrow (like its title), but for now it's an object.

Reply:
@soumyaloka that was my initial guess too, but I wanted to make sure. @ImCoolNowRight yeah im just concerned with making sure that the TS reflects the JS, do have a place where the object shape is defined?

Reply:
It's a whole Marquee component. The Row is just a bunch of Marquees as items, and the `selected` getter is just returning whatever Marquee we're on.